xwiki-platform-core/xwiki-platform-ratings/xwiki-platform-ratings-api/src/main/java/org/xwiki/ratings/internal/SeparatePageRating.java

```diff
@@ -212,7 +212,7 @@ public void save() throws RatingsException
             document.setContentDirty(false);
             document.setCreatorReference(superadmin);
             document.setAuthorReference(superadmin);
-            context.getWiki().saveDocument(getDocument(), context);
+            context.getWiki().saveDocument(getDocument(), context.getMessageTool().get("rating.saveComment"), true, context);
```

Comment:
This should have been `localizationManager.getTranslationPlain("rating.saveComment")` with `@Inject private ContextualLocalizationManager localizationManager;`.

MessageTool is deprecated since 4.3M2.

See https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-localization/xwiki-platform-localization-api/src/main/java/org/xwiki/localization/ContextualLocalizationManager.java